android/gradle/dependencies.gradle

```diff
@@ -16,7 +16,11 @@
 
 def versions = [
     dagger: "2.22.1",
-    support: '25.1.1',
+    androidx: [
+            annotations: '1.1.0',
```

Comment:
Nit: Can you make this 4 spaces instead?

Reply:
done